modules/ROOT/pages/access-logging.adoc

```diff
@@ -41,16 +41,19 @@ To enable logging for individual endpoints, use an `accessLogging` child element
 
 == HTTP access log format
 
-Use the `accessLogFormat` property to specify the information  and format you want to include in the NCSA access log for an HTTP transport channel. The value for this property is a space-separated list of options.
+Use the `accessLogFormat` property to specify the information and format you want to include in the NCSA access log for an HTTP transport channel. The value for this property is a space-separated list of options.
```

Comment:
Acrolinx picked this up for being too long.

Reply:
I see these sentences as being 26 and 11 words long- not too long.